compiler-rs/clients_schema/src/lib.rs

```diff
@@ -506,6 +506,7 @@ pub struct SchemaExample {
     pub value: Option<String>,
     pub external_value: Option<String>,
     pub alternatives: Option<Vec<ExampleAlternative>>,
+    pub alternative_java: Option<Vec<ExampleAlternative>>,
```

Comment:
Probably missing some context: why isn't that part of `alternatives`, since `ExampleAlternative` has a `language` property?

Reply:
I discussed this with Laura earlier today. We are not going to use a separate entry for Java, we can add it to the same list used by the other languages.

I have some changes in #4582 that she needs for this to work. Once my change is merged she'll be able to improve this and integrate Java fully with the other languages.